appveyor.yml

```diff
@@ -27,8 +27,8 @@ environment:
 install:
   - git submodule update --init
   - rmdir C:\go /s /q
-  - appveyor DownloadFile https://dl.google.com/go/go1.13.windows-%APP_ARCH%.zip
-  - 7z x go1.13.windows-%APP_ARCH%.zip -y -oC:\ > NUL
+  - appveyor DownloadFile https://dl.google.com/go/go1.14.windows-%APP_ARCH%.zip
```

Comment:
this should actually be `https://dl.google.com/go/go1.14.1.windows-%APP_ARCH%.zip`

Reply:
or does this URL guarantee the latest version?

Reply:
That URL is for the latest version, and I would keep it to match the others.